docs/UserGuide.md

```diff
@@ -257,7 +289,7 @@ In which case, if the user wants to fix the data JSON, the user should exit the
 | **Exit**       | `exit`                                                                                                                                                  |
 | **Add**        | `add n/NAME e/EMAIL g/GITHUB tl/TELEGRAM_HANDLE i/STUDENT_ID [t/TAG]...` <br> e.g., `add n/James Ho e/jamesho@email.com g/jamesH t/jamesho i/A0123456T` |
 | **List**       | `list`                                                                                                                                                  |
-| **Edit**       | `edit INDEX [n/NAME] [e/EMAIL] [g/GITHUB] [tl/TELEGRAM_HANDLE] [i/STUDENT_ID] [t/TAG]...`<br> e.g.,`edit 2 n/James Lee g/jamesHo`                       |
+| **Edit**       | `edit INDEX (n/NAME) (e/EMAIL) (g/GITHUB) (tl/TELEGRAM_HANDLE) (i/STUDENT_ID) (t/TAG)...`<br> e.g.,`edit 2 n/James Lee g/jamesHo`                       |
```

Comment:
Sorry why did you change this from `[` to `(` ?. From what i understand anything between `[ ]` is optional which is the expected behavior of the `edit` command.